build-helper/less.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
```

Comment:
This necessary? Just curious - I thought all parsing of the JS files was jshinted - or is that only the application code?